Kudu.Contracts/Jobs/JobSettingsKeys.cs

```diff
@@ -6,5 +6,6 @@ public static class JobSettingsKeys
         public const string StoppingWaitTime = "stopping_wait_time";
         public const string IsInPlace = "is_in_place";
         public const string Schedule = "schedule";
+        public const string IsWebHook = "is_webhook";
```

Comment:
I think it's pretty cheap. I think the thinking was that the wide majority of webjobs won't need this, so there is no point in doing this for every continuous webjob but I do agree that not having a setting is more convenient, I'll remove it.